<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="ProjectsTest"
           tests="2"
           failures="2"
           errors="0"
           skipped="0"
           time="0.000">
  <properties />
            <testcase name="logOutProfile" time="0.000" classname="org.fundacion.automation.projects.CreatePublicProjectTest">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  org.fundacion.automation.projects.CreatePublicProjectTest.logOutProfile(CreatePublicProjectTest.java:51)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
  org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
  org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:786)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
  org.testng.SuiteRunner.run(SuiteRunner.java:254)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
  org.testng.TestNG.run(TestNG.java:1057)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
      </failure>

          </testcase>
            <testcase name="testCreatePublicProject" time="0.000" classname="org.fundacion.automation.projects.CreatePublicProjectTest">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  org.fundacion.automation.projects.CreatePublicProjectTest.testCreatePublicProject(CreatePublicProjectTest.java:28)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
  org.testng.SuiteRunner.run(SuiteRunner.java:254)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
  org.testng.TestNG.run(TestNG.java:1057)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
